content_scripts/vimium_frontend.coffee

```diff
@@ -461,26 +461,20 @@ extend window,
 # Track which keydown events we have handled, so that we can subsequently suppress the corresponding keyup
 # event.
 KeydownEvents =
-  handledEvents: {}
-
-  stringify: (event) ->
-    JSON.stringify
-      metaKey: event.metaKey
-      altKey: event.altKey
-      ctrlKey: event.ctrlKey
-      keyIdentifier: event.keyIdentifier
-      keyCode: event.keyCode
+  handledEvents: new Uint8Array(256)
```

Comment:
Why use this rather than a normal object? I don't think we're massively concerned about the speed of this function, and this makes it brittle to future changes.